modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4504,6 +4469,8 @@ private void _validateValues(
 			return;
 		}
 
+		ObjectDefinition objectDefinition = objectField.getObjectDefinition();
+
```

Comment:
@devpedrotavares so we don't need this, you can pass the `objectDefinition` to the `_validateValues` method